interface/reports/username_report.php

```diff
@@ -0,0 +1,251 @@
+<?php
+/**
+ * interface/reports/claims_viewer.php List number of documents.
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ * Copyright (c) 2018 Growlingflea Software <daniel@growlingflea.com>
+ * File adapted for user activity log.
+ * @package ppc-db
+ * @author  Tony
+ */
+ $fake_register_globals=false;
+ $sanitize_all_escapes=true;
+
+require_once("../globals.php");
+require_once("$srcdir/sql.inc");
+require_once("$srcdir/formatting.inc.php");
+require_once("$srcdir/vendor/libreehr/Framework/DataTable/DataTable.php");
+require_once( "reports_controllers/AppointmentsController.php");
+
+
+
+?>
+<head>
+<?php html_header_show();?>
+<title><?php xl('User Activity Report','e'); ?></title>
+<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<style type="text/css">
+@import "<?php echo $GLOBALS['webroot'] ?>/assets/js/datatables/media/css/demo_page.css";
+@import "<?php echo $GLOBALS['webroot'] ?>/assets/js/datatables/media/css/demo_table.css";
+@import "<?php echo $GLOBALS['webroot'] ?>/assets/js/css/jquery-ui-1-12-1/jquery-ui.css";
+<!-- @import "<?php echo $GLOBALS['webroot'] ?>/library/css/jquery.tooltip.css"; -->
+.mytopdiv { float: left; margin-right: 1em; }
+</style>
+
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/assets/js/datatables/media/js/jquery.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/assets/js/datatables/media/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery-ui-1.8.21.custom.min.js"></script>
```

Comment:
Is this version required or will 3.1.1 work

Reply:
Fixed